pkg/upstream/http.go

```diff
@@ -92,6 +95,19 @@ func (h *httpUpstreamProxy) ServeHTTP(rw http.ResponseWriter, req *http.Request)
 	}
 }
 
+type unixRoundTripper struct {
```

Comment:
Please add an appropriate godoc to the struct and fields within it

Reply:
done :)